app/views/periodic_expenses/new.slim

```diff
@@ -1,16 +1,22 @@
 = simple_form_for @periodic_expense do |form|
-  = form.input :name
+  = form.input :start_date
 
-  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
+  = form.input :end_date
 
-  = form.input :amount
+  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
```

Comment:
I moved this lines around and didn't notice it. Shame on me. :disappointed: 
Moved it to controller.